### ISSUE TITLE
On changing feature offer features that are likely retagging targets 

### ISSUE THREAD

**Issue by matkoniecz** [openstreetmap/iD#6552]
For example "secondary road" on using change feature may display "Road closed" (`highway=construction`) as the first on the list.

Inspired by https://www.openstreetmap.org/user/Lars%20ST3422/diary/368843

**Comment by quincylvania** [comment]
Good idea. It makes sense to prioritize similar features when changing a preset. This would go well with iD v3 and other schema-aware improvements like #4173.
